content/posts/newsletter-018/index.md

```diff
@@ -230,6 +230,31 @@ which will be released in the Steam.
 [Station Iapetus]: https://github.com/mrDIMAS/StationIapetus
 [si-youtube]: https://www.youtube.com/watch?v=JCH2U5JOMlU
 
+### [Dwarf world][dwarf-world]
+
+![Dwarf world](dwarfworld_lights.gif)
+
+[Dwarf World] by [Alex Ene][@_AlexEne_] has added a couple of updates and
```

Comment:
The URLs for these links are missing.

Reply:
They are mentioned above in the podcast section and I thought it's not needed to add them again

Reply:
Oh sorry, you're totally right - one of the links seems to be broken, but I can fix that via GitHub UI :+1: